uber/templates/fields/attendee.html

```diff
@@ -1175,14 +1177,14 @@ <h4>{{ tier.name }}</h4>
 {% endset %}
 
 
-{% set pii_consent %}
+{% set pii_consent_checkbox %}
```

Comment:
We don't actually store `pii_consent` as a column on the Attendee table, so we instead use a page-level variable called `pii_consent` to keep the box checked or unchecked. Calling this field `pii_consent` overrode that variable, so the box wasn't being pre-checked properly!